### DISCUSSION TITLE
Welcome to awesome-portfolio-websites Discussions!

### DISCUSSION THREAD

**Discussion by smaranjitghose** [smaranjitghose/awesome-portfolio-websites#308] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Vishal-raj-1** [comment]
Hello Everyone !! I am Vishal , a frontend developer.  I want to contribute to open source and learn as many things I can.

**Comment by Vishal-raj-1** [comment]


**Reply by smaranjitghose** [reply]
By the end of this month. Although if you wish, you can start working in parallel on a separate branch

**Comment by jainh6148** [comment]


**Reply by smaranjitghose** [reply]
This project is open for everyone to contribute @jainh6148 .  Raise a new issue or get assigned to an already raised one, discuss with the maintainers, give a pr, incorporate feedback(if any) and get it merged. As simple as that

**Comment by Swastik2000** [comment]
@smaranjitghose I am a beginner. Can you please tell me? How you are using social media icons like GitHub medium etc with importing them. This means if in future we want to add a new icon for CodeChef profile then how can I do that.
